packages/runtime-core/src/componentProps.ts

```diff
@@ -795,6 +797,8 @@ function styleValue(value: unknown, type: string): string {
     return `"${value}"`
   } else if (type === 'Number') {
     return `${Number(value)}`
+  } else if (isSymbol(value)) {
+    return stringifySymbol(value)
```

Comment:
I think this check needs to come first, otherwise it'll still fail when a Symbol is passed for a prop with `type: String`.